src/js/navitron.js

```diff
@@ -135,7 +145,13 @@
                             $pane.find(selectors.CONTENT).attr('tabindex', 0).focus();
 
                             plugin._trigger('onShown', { pane: $pane });
-                        });
+                        };
+
+                        if (typeof window.requestAnimationFrame === 'function') {
```

Comment:
Would it be bad to monkeypatch in `window.requestAnimationFrame` in the test harness so we don't have to put these all over our production code?